$NetBSD: patch-meson.build,v 1.1 2025/10/31 12:29:12 jperkin Exp $

Disable libsocket handling on SunOS.  meson cannot handle the fact that this
library is not provided by pkg-config and refuses to build gdk-pixbuf-loader.
We already just link against it directly anyway.

--- meson.build.orig	2026-02-13 21:19:01 UTC
+++ meson.build
@@ -180,7 +180,7 @@ m_dep = cc.find_library('m', required: f
 # into '-lsocket' and '-lnsl'.  Anything that calls socketpair(), getifaddr(),
 # etc. probably needs to include network_deps
 #
-if host_system == 'sunos'
+if host_system == 'meson-dependencies-are-broken'
   network_deps = [
     cc.find_library('socket', required: false),
   ]
